snakemake/logging.py

```
@@ -305,12 +243,504 @@ def log_handler(self, msg):
+
+    def can_color_tty(self, mode):
+        """
+        Check if the terminal supports colors.
+        """
+        from snakemake_interface_executor_plugins.settings import ExecMode
+
+        if "TERM" in os.environ and os.environ["TERM"] == "dumb":
+            return False
+        if mode == ExecMode.SUBPROCESS:
+            return True
+        return self.is_tty and not platform.system() == "Windows"
```

Ruff

Use `platform.system() != "Windows"` instead of `not platform.system() == "Windows"`

Replace with `!=` operator

(SIM201)